src/ui/public/courier/saved_object/saved_object.js

```diff
@@ -18,6 +18,28 @@ import MappingSetupProvider from 'ui/utils/mapping_setup';
 
 import DocSourceProvider from '../data_source/admin_doc_source';
 import SearchSourceProvider from '../data_source/search_source';
+import { getTitleAlreadyExists } from './get_title_already_exists';
+
+/**
+ * An error message to be used when the user rejects a confirm overwrite.
+ * @type {string}
+ */
+const OVERWRITE_REJECTED = 'Overwrite confirmation was rejected';
+/**
+ * An error message to be used when the user rejects a confirm save with duplicate title.
+ * @type {string}
+ */
+const SAVE_DUPLICATE_REJECTED = 'Save with duplicate title confirmation was rejected';
+
+/**
+ * Returns true if the error is expected and should be suppressed (e.g. not shown to the user).
+ * @param error {Error} the error
+ * @return {boolean}
+ */
+function getShouldSuppressExpectedError(error) {
```

Comment:
How about renaming this to `isErrorNonFatal`? This way the name of function helps us assign meaning to the error without specifying what we should do about it. If for some reason we decide to return the error instead of suppressing it, this function will still be useful and won't need to be renamed.